doc/modules/ROOT/pages/projects.adoc

```diff
@@ -55,6 +55,12 @@ are the configuration files of various build tools. Out of the box the following
 | SConstruct
 | Scons project file
 
+| Nix
+| Nix project file
+
+| Nix Flake
```

Comment:
Here it should be the file name (`flake.nix`) and above as well `default.nix`.

Reply:
Oh right, thanks! I force pushed the changes.